src/web/app/page.component.ts

```diff
@@ -75,6 +75,9 @@ export class PageComponent implements OnInit {
         });
       }
     });
+    if (environment.frontendUrl) {
+      this.logoutUrl += `?frontendUrl=${environment.frontendUrl}`;
+    }
```

Comment:
Don't think you need this anymore?

Reply:
Sorry these were added due to the rebase I think. Will remove them.

Reply:
Uh-oh, this should not have been removed.